daisy/test_data/test_no_project_no_zone.wf.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "some-name",
+  "region": "us-central1",
+  "gcsPath": "gs://some-bucket/images",
+  "oauthPath": "somefile",
+  "vars": {
+    "bootstrap_instance_name": {"Value": "bootstrap-${NAME}", "Required": true},
+    "machine_type": "n1-standard-1",
+    "key1": "var1", 
+    "key2": "var2"
+  },
+  "steps": {
+    "create-disks": {
+      "createDisks": [
+        {
+          "Name": "bootstrap",
+          "SourceImage": "projects/windows-cloud/global/images/family/windows-server-2016-core",
+          "SizeGb": "50",
+          "Type": "pd-ssd"
+        },
+        {
+          "Name": "image",
+          "SourceImage": "projects/windows-cloud/global/images/family/windows-server-2016-core",
```

Comment:
is there any booting problem when using 2 identical Windows images? I remember it is a problem for linux.

Reply:
No, each windows image is unique

Reply:
This is a test file used only for parsing verification. Even if it was a workflow that we actually run, it's fine to create two disks from one image. I assume it would be problematic to use a single disk on two VMs.